server/gameconnection.py

```diff
@@ -163,10 +164,18 @@ def send_message(self, message):
         result = await self.EstablishConnection(peer)
         if not result:
             self.abort("Failed connecting to host {}".format(peer))
+
+        try:
+            await asyncio.wait_for(self.game.joinable, 60.0)
```

Comment:
Yeah, I was thinking about that. Isn't EstablishConnection all about the connection setup which is separate from needing the actual game to be listening, it's all handled by the other players lobby-client?

When the connection handshake is done we need to wait for Zlo's slow ForgedAlliance.exe to actually be prepared for the JoinGame command.